specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -304,7 +306,10 @@ implementation includes support for a technology marked as Optional,
 that technology must be supported in the containers specified above. If
 a product implementation does not support a technology marked as
 Optional, it must not include the APIs for that
-technology.<<a3665, 18>>
+technology.footnote:[Note that a component specification is permitted to specify
+an exception to this in order to accommodate interface type dependencies—for example,
+the Jakarta™ Enterprise Beans SessionContext dependency on the
+javax.xml.rpc.handler.MessageContext type.]
```

Comment:
this is a really minor nit, but I'd probably mark javax.xml.rpc.handler.MessageContext up using __ as types are elsewhere.

Reply:
Nice catch.  I just did a cut-n-paste from the existing footnotes.  Thanks.